checkov/cloudformation/graph_builder/variable_rendering/renderer.py

```diff
@@ -0,0 +1,206 @@
+            value=val_to_eval, vertices_block_name_map=vertices_block_name_map
+        )
+        if not referenced_vertices:
+            # DependsOn or Condition connections
+            pass
+
+        if referenced_vertices:
+            # Ref, GetAtt, FindInMap, If, Sub connections
+            cfn_evaluation_function = None
+            for curr_evaluation_function in self.EVALUATION_CFN_FUNCTIONS:
+                if curr_evaluation_function in val_to_eval:
+                    cfn_evaluation_function = curr_evaluation_function
+            if cfn_evaluation_function:
+                original_value = val_to_eval.get(cfn_evaluation_function, None)
+
+                for edge in edge_list:
+                    dest_vertex_attributes = self.local_graph.get_vertex_attributes_by_index(edge.dest)
+                    evaluated_value = self.evaluation_methods[cfn_evaluation_function](val_to_eval[cfn_evaluation_function], dest_vertex_attributes)
+                    if evaluated_value and evaluated_value != original_value:
+                        val_to_eval[cfn_evaluation_function] = evaluated_value
+                        self.update_evaluated_value(
+                            changed_attribute_key=edge.label,
+                            changed_attribute_value=evaluated_value,
+                            vertex=edge.origin,
+                            change_origin_id=edge.dest,
+                            attribute_at_dest=edge.label,
+                        )
+
+    @staticmethod
+    def _evaluate_ref_connection(value: str, dest_vertex_attributes) -> Optional[str]:
+        evaluated_value = None
+
+        # in case of Ref we take only Parameter's default value
+        if value == dest_vertex_attributes.get(CustomAttributes.BLOCK_NAME) and dest_vertex_attributes.get(
+                CustomAttributes.BLOCK_TYPE) == BlockType.PARAMETERS:
+            evaluated_value = dest_vertex_attributes.get('Default', None)
+            if evaluated_value:
+                evaluated_value = str(evaluated_value)
+
+        return evaluated_value
+
+    @staticmethod
+    def _evaluate_findinmap_connection(value: List[str], dest_vertex_attributes) -> Optional[str]:
+        evaluated_value = None
+
+        # value = [ "MapName", "TopLevelKey", "SecondLevelKey"]
+        if isinstance(value, list) and len(value) == 3:
+            map_name = value[0]
+            top_level_key = value[1]
+            second_level_key = value[2]
+
+            if all(isinstance(element, str) for element in value) and \
+                    map_name == dest_vertex_attributes.get(CustomAttributes.BLOCK_NAME) and \
+                    dest_vertex_attributes.get(CustomAttributes.BLOCK_TYPE) == BlockType.MAPPINGS:
+                evaluated_value = dest_vertex_attributes.get(f'{top_level_key}.{second_level_key}', None)
+                if evaluated_value:
+                    evaluated_value = str(evaluated_value)
+
+        return evaluated_value
```

Comment:
this could also be rewritten similar to my suggestion in `_evaluate_ref_connection()`